packages/@react-native-windows/codegen/src/Cli.ts

```diff
@@ -6,26 +6,16 @@
  */
```

Comment:
sidebar, are/should there be tests for this codegen package?

Reply:
We run it on the complete set of >60 native modules within core -- which should provide a certain amount of coverage.  -- The output of that is in vnext/codegen, so we can monitor for changes to the codegen'd output there.

If the output changes in a bad way it would likely break one or more of RNW's module implementations.